kyo-prelude/shared/src/test/scala/kyo2/AbortTest.scala

```diff
@@ -316,8 +316,8 @@ class AbortsTest extends Test:
                     val a = kyo2.Abort.run[Distinct1](distinct)
                     val b = kyo2.Abort.run[Distinct2](a).eval
                     assert(b == Result.success(Result.error(d1)))
-
-                    val c = kyo2.Abort.run(distinct).eval
+                    assertDoesNotCompile("kyo2.Abort.run[Distinct1 | Distinct2](distinct)")
```

Comment:
Another scenario that was incorrect. The `ClassTag` was getting widened to `Throwable`, which would catch aborts not specified in the type union.